src/docs/sphinx/ARM.rst

```diff
@@ -4,6 +4,70 @@
 .. # SPDX-License-Identifier: MIT
 
 
-ARM Overview (under development)
+Arm Overview
```

Comment:
Should we use ARM or Arm? The code is using ARM, but docs are using Arm.

Reply:
Good point. ARM (Advanced RISC Machines) refers to the architecture whereas Arm Ltd. is the company that built Arm Juno r2 board. Since the reference documentation for the Arm Juno r2 board points to Arm Ltd. website, I used Arm everywhere in the documentation. I think a better approach would be to use 'Arm' when referring to the Arm Juno r2 board and 'ARM' when referring to the architecture. I'll change the documentation accordingly. 

This specific line should say 'ARM overview' instead of 'Arm Overview' so I'll make that change as well.